server.go

```diff
@@ -76,9 +75,18 @@ const (
 	// maxProtocolVersion is the max protocol version the server supports.
 	maxProtocolVersion = wire.InitStateVersion
 
-	// maxKnownAddrsPerPeer is the maximum number of items to keep in the
-	// per-peer known address cache.
+	// maxKnownAddrsPerPeer is the maximum number of items to track in the
+	// per-peer known address filter.
+	//
+	// knownAddrsFPRate is the false positive rate to use for the APBF used to
+	// track known addresses per-peer.  It is set to a rate of 1 per 1000 since
+	// addresses are not very large and they only need to be filtered once per
+	// connection, so an extra 10 of them being sent (on average) again even
+	// though they technically wouldn't need to is a good tradeoff.
+	//
+	// These values result in about 40 KiB memory usage including overhead.
 	maxKnownAddrsPerPeer = 10000
+	knownAddrsFPRate     = 0.001
```

Comment:
`maxKnownAddrsPerPeer` is now disjointed from its description.

Reply:
I've updated the comments to treat them as a group as they are intertwined.